x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_embaddable.tsx

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
fyi, filename has a typo - `embaddable.tsx`  -> `embeddable.tsx`

Reply:
Fixed in https://github.com/elastic/kibana/pull/64056/commits/7769bd0d22fcd49c6a201d33aaa5fdc5ab3be522